README.md

```diff
@@ -1,5 +1,9 @@
 # A collection of Elliptic Curves for ZkCrypto traits
 
+[![crates.io version](https://img.shields.io/crates/v/halo2curves.svg)](https://crates.io/crates/halo2curves)
+[![docs.rs availability](https://img.shields.io/docsrs/halo2curves?label=docs.rs)](https://docs.rs/halo2curves/)
+[![Build status](https://img.shields.io/github/actions/workflow/status/privacy-scaling-explorations/halo2curves/ci.yml?branch=main)](https://github.com/privacy-scaling-explorations/halo2curves//actions)
```

Comment:
Just a nitpick that could be ignored (the link is still working with double slash)

```suggestion
[![Build status](https://img.shields.io/github/actions/workflow/status/privacy-scaling-explorations/halo2curves/ci.yml?branch=main)](https://github.com/privacy-scaling-explorations/halo2curves/actions)
```

Reply:
Reflected in 428ac24